### PR TITLE
[OPS-1695] add `hstm-jb-core` which is like `hstm-core` but uses fruition instead of hstm-juicebox

### PR DESCRIPTION
**Pull request by radix**
Ticket: [OPS-1695](https://juiceanalytics.atlassian.net/browse/OPS-1695)
Type: Feature

## Changes

This involved reworking some of the `jb` code to avoid setting the `AWS_PROFILE` to `HSTM` until *after* the `docker pull` command is run, so we can still pull the image using regular JB legacy credentials, and then run it with HSTM credentials.

### CODE REVIEW

**Comment by cwire4** [comment]
Hmm, this will probably be fine for our developers working on HSTM, but I think permissions will still need to change for this to work for only HSTM developers (I'm pretty sure that number isn't 0 at least).

**Comment by radix** [comment]
@cwire4 you're right, and I just realized that this will probably also break the `hstm-core` env (and others) since it will try to fetch the controlcenter image from the HSTM account before switching to the HSTM profile... I'll have to figure out a work around for this.

**Comment by cwire4** [comment]
Ah, good point!

File: jbcli/jbcli/cli/jb.py [hunk 1]
```diff
@@ -548,3 +548,8 @@ def search(username, password, env, data_service_log,
 
     if lookback_window > 90:
         print("There is only 90 days of history retained")
+
+
+def activate_hstm():
```
Comment (chrisgemignani) [inline jbcli/jbcli/cli/jb.py]: this is cute 🐶

**Comment by radix** [comment]
Yeah, I think `jb start core` and `jb start hstm-core` would be a lot nicer.

**Comment by radix** [comment]
closing this because I need to redo it

**Pull request closed without merging**